partition/roles/systemd-networkd/tasks/main.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-- include_tasks: redhat.yaml
+- import_tasks: redhat.yaml
```

Comment:
Why `import_` instead of `include_`?

Reply:
Import enables Ansible to compile the task execution at the start, include adds them dynamically during the execution. If there is a syntax error in `redhat.yaml` for example, Ansible will fail immediately at the start when using `import` instead of dying in the middle of the run when the include happens, which I think is beneficial.

Here are some more tradoffs: https://docs.ansible.com/ansible/2.9/user_guide/playbooks_reuse.html#tradeoffs-and-pitfalls-between-includes-and-imports